.github/workflows/build-validation.yml

```diff
@@ -2,7 +2,6 @@
 name: Snippets 5000
 
 # Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the master branch
```

Comment:
Did you mean to delete the entire line?

Reply:
Yes, the config isn't set for master only